github/event_types.go

```diff
@@ -190,6 +190,38 @@ type IssueCommentEvent struct {
 	Sender  *User       `json:"sender,omitempty"`
 }
 
+// Triggered when a repository's label is created, edited, or deleted.
+// The Webhook event name is "label"
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#labelevent
+type LabelEvent struct {
+	//Action is the action that was performed. Possible values are:
+	// "created", "edited", "deleted"
+	Action *string `json:"action,omitempty"`
+	Label  *Label  `json:"label,omitempty"`
+
+	// The following fields are only populated by Webhook events.
```

Comment:
Isn't this entire struct only for Webhook events, or are the other fields above used elsewhere somehow?